bench-tps/src/bench.rs

```diff
@@ -577,18 +586,18 @@ fn should_switch_directions(num_lamports_per_account: u64, i: u64) -> bool {
     i % (num_lamports_per_account / 4) == 0 && (i >= (3 * num_lamports_per_account) / 4)
 }
 
-pub fn generate_keypairs(seed_keypair: &Keypair, count: u64) -> Vec<Keypair> {
+pub fn generate_keypairs(seed_keypair: &Keypair, count: u64) -> (Vec<Keypair>, u32) {
     let mut seed = [0u8; 32];
     seed.copy_from_slice(&seed_keypair.to_bytes()[..32]);
     let mut rnd = GenKeys::new(seed);
 
     let mut total_keys = 0;
-    let mut counter = 1;
+    let mut counter = 0;
     while total_keys < count {
-        total_keys += MAX_SPENDS_PER_TX.pow(counter);
         counter += 1;
+        total_keys += MAX_SPENDS_PER_TX.pow(counter);
```

Comment:
I know it's not performance sensitive code, but the pow every iteration makes me cringe. Can just keep the previous loop iteration value around, if you know x^(n-1) then computing x^(n) is just 1 multiply.